bundles/CoreBundle/src/DependencyInjection/PimcoreCoreExtension.php

```diff
@@ -125,6 +127,13 @@ public function loadInternal(array $config, ContainerBuilder $container): void
         $container->setParameter('pimcore.workflow', $config['workflows']);
 
         $this->addContextRoutes($container, $config['context']);
+
+        $container->registerAttributeForAutoconfiguration(
+            AsAreabrick::class,
+            static function (ChildDefinition $definition, AsAreabrick $attribute): void {
+                $definition->addTag('pimcore.area.brick', ['id' => $attribute->id]);
```

Comment:
It doesn't work when using the attribute (with or without id) as defined in the description. getting this error:
<img width="541" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/ab0d3cf0-df74-4a36-953e-1ffad3b34129">

Reply:
@jdreesen we are reviewing this PR now to merge it. Could you please confirm this issue? thanks!

Reply:
Right, the last commit was broken. I fixed and force-pushed it. Should work now.

Note that I'm on vacation now until August 16.

Reply:
@jdreesen thanks Jacob!! I'll review it soon. enjoy your vacation 😊